tests/functional/aws-node-sdk/test/multipleBackend/utils.js

```diff
@@ -61,44 +61,34 @@ const utils = {
 utils.uniqName = name => `${name}${new Date().getTime()}`;
 
 utils.getAzureClient = () => {
-    let isTestingAzure;
-    let azureBlobEndpoint;
-    let azureBlobSAS;
-    let azureClient;
-    if (process.env[`${azureLocation}_AZURE_BLOB_ENDPOINT`]) {
-        isTestingAzure = true;
-        azureBlobEndpoint = process.env[`${azureLocation}_AZURE_BLOB_ENDPOINT`];
-    } else if (config.locationConstraints[azureLocation] &&
-          config.locationConstraints[azureLocation].details &&
-          config.locationConstraints[azureLocation].details.azureBlobEndpoint) {
-        isTestingAzure = true;
-        azureBlobEndpoint =
-          config.locationConstraints[azureLocation].details.azureBlobEndpoint;
-    } else {
-        isTestingAzure = false;
-    }
+    const params = {};
+    const envMap = {
+        azureStorageAccountName: 'AZURE_STORAGE_ACCOUNT_NAME',
+        azureStorageAccessKey: 'AZURE_STORAGE_ACCESS_KEY',
+    };
 
-    if (isTestingAzure) {
-        if (process.env[`${azureLocation}_AZURE_BLOB_SAS`]) {
-            azureBlobSAS = process.env[`${azureLocation}_AZURE_BLOB_SAS`];
-            isTestingAzure = true;
-        } else if (config.locationConstraints[azureLocation] &&
+    const isTestingAzure = Object.keys(envMap).every(key => {
```

Comment:
Maybe you can just rely on `azureStorageAccountName` presence, since an azure test would not make sense without the access key set as well.

Reply:
We also create the blobservice client below, and we need to get the azureStorageAccessKey (stored in params in the every step) for that.